### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by bobbrow**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ### New Features
 * Add an 'Extract to Function' (or Member Function) code action after selecting code. [#1162](https://github.com/microsoft/vscode-cpptools/issues/1162)
   * Currently, it's only enabled when `C_Cpp.experimentFeatures` is `true`. Also, 'Extract to Free Function' is disabled.
-* Add the option to install a compiler, accessible through the walkthrough. [PR #11286](https://github.com/microsoft/vscode-cpptools/pull/11286)
+* Add a command to install a compiler, accessible through the walkthrough and Select IntelliSense Configuration experience. [#10525](https://github.com/microsoft/vscode-cpptools/issues/10525)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: We had "a command" previously, but @AlexandraKemperMS said to use "the option".

`and Select IntelliSense Configuration experience` should be `and the 'Select IntelliSense Configuration' experience`.

What is the point of updating the changelog right now? We normally only do that before a release. Can you update our "internal" changelog instead, and/or update the "live" release page at https://github.com/microsoft/vscode-cpptools/releases/tag/v1.18.0 .

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (bobbrow) [reply Extension/CHANGELOG.md]: The point of updating it is so that it's correct in the next release.

Comment (bobbrow) [reply Extension/CHANGELOG.md]: You're right about the option bit though. I forgot that this command is not accessible via the command palette.

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 ### New Features
 * Add an 'Extract to Function' (or Member Function) code action after selecting code. [#1162](https://github.com/microsoft/vscode-cpptools/issues/1162)
   * Currently, it's only enabled when `C_Cpp.experimentFeatures` is `true`. Also, 'Extract to Free Function' is disabled.
-* Add the option to install a compiler, accessible through the walkthrough. [PR #11286](https://github.com/microsoft/vscode-cpptools/pull/11286)
+* Add an option to install a compiler, accessible through the walkthrough and the Select IntelliSense Configuration experience. [#10525](https://github.com/microsoft/vscode-cpptools/issues/10525)
```
Comment (Colengms) [inline Extension/CHANGELOG.md]: We noticed that it isn't accessible in Select IntelliSense Configuration on Windows. Do you want to be more concise about that?  :)

Comment (bobbrow) [reply Extension/CHANGELOG.md]: I don't think we want to go into all the specifics in the changelog.  How about I shorten it to "Compiler acquisition improvements" and update the Issue to make sure it covers the full behvior?

**Review by sean-mcmanus — APPROVED** [review]
This is fine, but you

**Review by Colengms — APPROVED** [review]

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**